cmd/zb/main.go

```diff
@@ -59,8 +59,8 @@ func NewPerfRootCmd() *cobra.Command {
 		"Number of multiple requests to make at a time")
 	rootCmd.Flags().IntVarP(&requests, "requests", "n", 1,
 		"Number of requests to perform")
-	rootCmd.Flags().StringVarP(&output, "output-format", "o", "",
-		"Output format of test results [default: stdout]")
+	rootCmd.Flags().StringVarP(&outFmt, "output-format", "o", "",
+		"Output format of test results: stdout (default), json, ci-cd")
```

Comment:
This is a bit confusing as there are not 3 options (stdout, json, ci-cd). And the stdout option is always enabled even if not specified.

Reply:
"ci-cd" is specific to the ci-cd we are using and only outputs "requests per sec" in the cicdSummary format.
json format instead would have everything in JSON format, which will be separate.